universal-application-tool-0.0.1/app/models/Applicant.java

```diff
@@ -1,35 +1,42 @@
 package models;
 
-import io.ebean.annotation.DbJsonB;
-import io.ebean.text.json.EJson;
+import com.jayway.jsonpath.DocumentContext;
+import com.jayway.jsonpath.JsonPath;
+import io.ebean.annotation.DbJson;
 import java.io.IOException;
-import java.util.Map;
-import javax.persistence.Entity;
-import javax.persistence.Table;
+import javax.persistence.*;
 import play.data.validation.Constraints;
+import services.applicant.ApplicantData;
 
 @Entity
-/** The ebeans mapped class that represents an individual applicant */
+/** The ebean mapped class that represents an individual applicant */
```

Comment:
Suggestion (non-blocking): javadoc comments should go before annotations since annotations are part of the thing being documented

Reply:
Done